### DISCUSSION TITLE
auto import suggestion is not working in version 7.5.6

### DISCUSSION THREAD

**Discussion by dante01yoon** [ReactiveX/rxjs#7055] [Q&A]
### Describe the bug

other libraries auto import works but only rxjs doesn't support auto import / suggestion in visual studio code. 

![스크린샷 2022-09-02 오전 10 44 58](https://user-images.githubusercontent.com/6510430/188041324-b4c3c964-2707-4226-894d-b941a933865f.png)

### Expected behavior

auto import should be showed on IDE
![스크린샷 2022-09-02 오전 10 46 53](https://user-images.githubusercontent.com/6510430/188041430-4f723a81-1d87-4807-94a1-6ff21e41d232.png)

### Version

7.5.6

**Comment by kwonoj** [comment]
I am not sure why this is related with rxjs core. I just created bare empty repo and vscode suggests this:

<img width="734" alt="image" src="https://user-images.githubusercontent.com/1210596/188042810-ed0a3585-870c-4fb4-b9bd-137206e0336a.png">

**Comment by voliva** [comment]
I've noticed this on my project too... except sometimes it works, sometimes it doesn't.

Still unsure what's causing it, or whether it's a configuration issue.
